core/src/main/java/io/cucumber/core/options/RuntimeOptionsParser.java

```diff
@@ -105,6 +105,9 @@ RuntimeOptionsBuilder parse(List<String> args) {
                     throw new CucumberException("--count must be > 0");
                 }
                 parsedOptions.setCount(count);
+            } else if (arg.equals("--object-factory")) {
+                String objectFactoryClassName = args.remove(0).trim();
```

Comment:
Is the trim needed?

Reply:
Probably not ... I removed it. Just trying to be overly safe ;-)